requirements.txt

```diff
@@ -7,20 +8,29 @@ dj-database-url==0.4.2
 Django==2.0.1
 django-debug-toolbar==1.9.1
 django-dotenv==1.4.2
+django-jet==1.0.7
 django-nose==1.4.5
 djangorestframework==3.7.7
 docopt==0.6.2
 flake8==3.5.0
 gunicorn==19.7.1
 idna==2.6
+lazy-object-proxy==1.3.1
```

Comment:
@nelmandela let's remove the other added packages too since they are not specific to this task. That is Lines 18, 21, 25, 26, 27, 31,32, 36 in the requirement file. If they are still necessary, we could add that in another story (which serves to address that.

Reply:
@nelmandela let's remove lazy-object-proxy.

Reply:
on it